webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AutomaticReportsCleaner.scala

```diff
@@ -304,6 +335,50 @@ class AutomaticReportsCleaning(
   }
   deleter ! CheckLaunch
 
+
+  // cleaning log info is special, it's not a cron but an "every NN minutes"
+  val deleteLogReportPropertyName = "rudder.batch.reportsCleaner.deleteLogReport.TTL"
+  val deleteLogttl = {
+    def toInt(s: String, orig: String): Int = {
+      try {
+        s.toInt
+      } catch {
+        case ex: NumberFormatException =>
+          logger.warn(s"Impossible to read value for '${deleteLogReportPropertyName}': '${orig}'. It should be either an integer (minutes) or a number of run (NNx , with NN an integer). Defaulting to 2 runs")
+          2
+      }
+    }
+    // parse user string
+    if(deleteLogttlString.endsWith("x")) {
+      val r = toInt(deleteLogttlString.substring(0, deleteLogttlString.size - 1), deleteLogttlString)
+      val interval = AutomaticReportsCleaning.getMaxRunMinutes(ldap) match {
+        case Full(x) =>
+          logger.debug(s"Found maximun run interval: ${x} minutes")
+          x
+        case eb: EmptyBox =>
+          logger.error((eb ?~! s"Error when trying to get maximun run interval, defaulting to  5 minutes. Error was: ").messageChain)
+          5
+      }
+      interval * r
+    } else {
+      toInt(deleteLogttlString, deleteLogttlString)
```

Comment:
in this case, it default to 2 minutes if number is malformed

Reply:
right :/